youtube_dl/extractor/youtube.py

```diff
@@ -1072,6 +1072,9 @@ def add_dash_mpd(video_info):
         else:
             video_categories = None
 
+        m = re.findall(r'''<meta(?:\s+[a-zA-Z0-9:._-]+(?:=[a-zA-Z0-9:._-]+|="[^"]+"|='[^']+'))*?\s+property=['"]?og:video:tag['"]?(?:\s+[a-zA-Z0-9:._-]+(?:=[a-zA-Z0-9:._-]+|="[^"]+"|='[^']+'))*?\s+content=['"]?([^>'"]+?)['"]?\s*>'''
```

Comment:
I can't do that, because youtube returns multiple meta tags, one per keyword. _html_search_meta() uses re.search under the hood, so it only finds the first instance.